slides/ds101-worskhop.mdx

```diff
@@ -27,6 +27,26 @@ export const theme = { ...CDTheme };
 
 ---
 
+# Schedule
```

Comment:
This came here after rebase :-( Somehow it is in diff even though the master already has this content.

Reply:
git log says:

```
* 2f12b49 - (HEAD -> making-a-toast, origin/making-a-toast) Update slides/ds101-worskhop.mdx (19 hours ago) <Andrey Okonetchnikov>
* 7664983 - Schedule (19 hours ago) <Varya Stepanova>
* b2a4495 - Review changes (19 hours ago) <Varya Stepanova>
* 4a6994c - Ex: Making a Toast. #5 (19 hours ago) <Varya Stepanova>
*   74a3882 - (origin/master, origin/HEAD, master) Merge pull request #19 from component-driven/schedule (19 hours ago) <Varya Stepanova>
|\  

```